helm/cluster-cloud-director/values.schema.json

```
@@ -183,7 +183,23 @@
             "type": "string"
         },
         "kubectlImage": {
-            "type": "object"
+            "type": "object",
+            "title": "kubectl Image",
+            "description": "Used by cluster-shared library chart.",
+            "additionalProperties": {
```

Pretty sure this should be `properties`, as you are naming the properties specifically.

```suggestion
            "properties": {
```

---

yes